zillow_crawler/main.py

```diff
@@ -40,6 +44,7 @@ def main():
 
```

Comment:
Can you do something with pages as well?

Reply:
I am calculating the total. Do you want to print each URL crawled as well? As in, you print the addresses in listings. I can print the ULRs underneath for all that crawled is true? let me know